pkg/engine/cli.go

```diff
@@ -366,7 +368,9 @@ func (em *EngineMain) RunEngine(cmd *cobra.Command, args []string) (exitCode int
 		return
 	}
 
-	context := &EngineContext{}
+	context := &EngineContext{
+		GlobalTag: architectureEngineCfg.globalTag,
```

Comment:
This is kind of leaking AWS specifics into general contexts. I'm okay with this for now, but maybe throw a TODO or FIXME and/or ticket to genericize it.

Reply:
i mean we could say the concept of tag applies to other things as well, like we can take kubernetes objects via labels and i would assume gcp azure have a similar concept. we could name it something else if we want